### ISSUE TITLE
Cannot migrate non-`public` schema in postgres

### ISSUE THREAD

**Issue by evantahler** [actionhero/ah-sequelize-plugin#490]
Adding `schema: "public"` to the config does not create tables in the proper schema.  The SequelizeMeta table is in the proper schema, but none of the user-generated tables are.

**Comment by evantahler** [comment]
Answer: You will need both `searchPath`, `dialectOptions`, and `schema` in your config (from https://github.com/sequelize/sequelize/issues/10875).  `schema` is where the SequelizeMeta table will be, and `dialectOptions.prependSearchPath` and `searchPath` are about where the tables go:


```ts
{
      autoMigrate: true,
      logging: false,
      dialect: dialect as Dialect,
      port: parseInt(port),
      database: database,
      schema: "foo", // <- HERE
      searchPath: "foo", // <- HERE
      host: host,
      username: username,
      password: password,
      models: [join(__dirname, "..", "models")],
      migrations: [join(__dirname, "..", "migrations")],
      migrationLogLevel: "info",
      dialectOptions: { 
        prependSearchPath: true, // <- HERE
      },
}
```

**Comment by evantahler** [comment]
@nholmes3 can you give this a try?  If it works, can you add something about this to the readme?

Thanks!

**Comment by nholmes3** [comment]
> @nholmes3 can you give this a try? If it works, can you add something about this to the readme?
> 
> Thanks!

Hey @evantahler , these configuration options are already present in the readme. It is actually a bug in the sequelize code that causes this problem. I have created [a PR over in that project](https://github.com/sequelize/sequelize/pull/14634) that should fix the underlying problem. 

I was just curious if the patch code referenced in #488 could still be used as an effective workaround in the meantime. Based on your comments there, it seems some additional development/problem solving might be required to get that code back up and usable as a workaround.

It would definitely be nice to have something to use while waiting on my PR in sequelize to go through the release cycle, but it seems that updating this code might take just as long. 

I would be interested in collaborating if you have any immediate thoughts.

**Comment by evantahler** [comment]
Sorry for the delay, I'm on vacation 🌴 

Awesome work merging the fix into Sequelize! 

Sure, let's connect on LinkedIn - I'm https://linkedin.com/in/evantahler and we'll take it from there.

**Comment by nholmes3** [comment]
Hey @evantahler no worries!

I haven't managed to reach you on LinkedIn so I am falling back to this thread. I hope that's okay. 

To elaborate further on my above comment, I was able to get [a fix for the sequelize bug](https://github.com/sequelize/sequelize/pull/14665) included in version 6.21.1.  Grouparoo currently [uses version 6.17.0 ](https://github.com/grouparoo/grouparoo/blob/main/core/package.json#L66). 

I know that Grouparoo is no longer under active development, but I was wondering if it is at all possible to release one final version that upgrades sequelize to 6.21.1. This would allow for Grouparoo users to be able to provide custom schema names in the database that Grouparoo uses. I believe that this functionality is crucial for using Grouparoo effectively.

If there is anything that I can do to help with this process, I am more than happy to contribute in any way possible.

Thanks for your consideration!
